openedx/core/djangoapps/schedules/management/commands/send_recurring_nudge.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import print_function
+
+import datetime
+import logging
+
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+import pytz
+
+from edx_ace.utils.date import serialize
+from openedx.core.djangoapps.schedules.models import ScheduleConfig
+from openedx.core.djangoapps.schedules.tasks import recurring_nudge_schedule_hour
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+
+from edx_ace.recipient_resolver import RecipientResolver
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ScheduleStartResolver(RecipientResolver):
+    def __init__(self, site, current_date):
+        self.site = site
+        self.current_date = current_date.replace(hour=0, minute=0, second=0)
+
+    def send(self, day, override_recipient_email=None):
+        """
+        Send a message to all users whose schedule started at ``self.current_date`` - ``day``.
+        """
+        if not ScheduleConfig.current(self.site).enqueue_recurring_nudge:
+            return
+
+        try:
+            site_config = SiteConfiguration.objects.get(site_id=self.site.id)
```

Comment:
This logic is not completely straightforward to understand and seems like it doesn't belong within this function.  It may be better suited to create a helper function for this within `SiteConfiguration`.